src/cpufreq_policy.hpp

```diff
@@ -78,8 +78,8 @@ class CpufreqPolicy
         , fd_freq{ -1 } // opened after governor is changed
         , name{ policy_dir.filename() }
         , original_governor{ current_governor }
-        , min_frequency{ read_freq_file(policy_dir / "scaling_min_freq") }
-        , max_frequency{ read_freq_file(policy_dir / "scaling_max_freq") }
+        , min_frequency{ read_freq_file(policy_dir / "cpuinfo_min_freq") }
+        , max_frequency{ read_freq_file(policy_dir / "cpuinfo_max_freq") }
```

Comment:
I'd prefer to have the commit message as a comment here, it may not be clear why the `cpuinfo_...` files are used here otherwise.